lib/vanagon/platform/windows.rb

```diff
@@ -107,7 +107,11 @@ def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         cg_name = "ProductComponentGroup"
         dir_ref = "INSTALLDIR"
-        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} -ext WiXUtilExtension -ext WixUIExtension"
+        wix_extensions =  "-ext WiXUtilExtension -ext WixUIExtension"
+        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} #{wix_extensions}"
```

Comment:
I'm partial to removing this so people don't get confused when they're creating their project wix files as to where this comes from/gets set